src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -143,6 +186,8 @@ def __init__(
         dynamic_thresholding_ratio: float = 0.995,
         clip_sample_range: float = 1.0,
         sample_max_value: float = 1.0,
+        timestep_scaling: str = "leading",
```

Comment:
Maybe `sample_step_selection_mode` is better?

Reply:
True, `scaling` is not very well chosen.

`steps_spacing_mode` maybe? think we refer mostly to how the timestep values are spaced no?